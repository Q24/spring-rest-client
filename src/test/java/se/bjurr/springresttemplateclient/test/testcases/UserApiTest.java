package se.bjurr.springresttemplateclient.test.testcases;

import org.junit.Test;
import se.bjurr.springresttemplateclient.test.spec.api.UserApi;
import se.bjurr.springresttemplateclient.test.utils.BaseApiTest;

public class UserApiTest extends BaseApiTest<UserApi> {

  @Override
  public Class<UserApi> getSutClass() {
    return UserApi.class;
  }

  @Test
  public void getUserByName() {
    this.getSut().getUserByName("tomas");
    this.verify();
  }
}
