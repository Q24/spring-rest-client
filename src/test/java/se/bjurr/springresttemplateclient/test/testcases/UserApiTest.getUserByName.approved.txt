[ {
  "url" : "http://localhost:X/user/tomas",
  "method" : "GET",
  "headers" : {
    "Accept" : "application/json",
    "Connection" : "keep-alive"
  },
  "cookies" : { },
  "body" : ""
} ]